src/app/(profile)/(components)/ProfileList/ProfilesWithFilter.tsx

```diff
@@ -23,19 +23,37 @@ export const ProfilesWithFilter: React.FC<{
     availabilityFilter,
     locationFilter,
     jobSpecializationFilter,
+    searchTermFilter,
   } = useFilters()
   const filteredProfileData = (data || [])
     .filter(filterByPosition(jobSpecializationFilter))
     .filter(filterBySeniority(seniorityFilter))
     .filter(filterByLocation(locationFilter))
     .filter(filterByTechnology(technologyFilter))
     .filter(filterByAvailability(availabilityFilter))
+    .filter((profile) =>
+      profile.fullName.toLowerCase().includes(searchTermFilter.toLowerCase()),
```

Comment:
**nit:** we should move it to separate function, same as above, so: `filterByFullName`

Reply:
oh, there is such function already - let's just use it